common/exported_symbols_mac.sym

```diff
@@ -0,0 +1,6 @@
+# These symbols are looked up from within the executable at runtime and must
+# be exported in the dynamic symbol table.
```

Comment:
Consider adding a comment about why we have a different file here for mac than exported_symbols.sym.

Reply:
I added a comment in the GN build file.